packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -149,6 +149,21 @@ class ShaderMask extends SingleChildRenderObjectWidget {
 ///
 /// This effect is relatively expensive, especially if the filter is non-local,
 /// such as a blur.
+///
+/// The effect is applied to all existing painted content. To constrain
```

Comment:
@collinjackson that's what I seem to have understood from our conversation. Or I might have gotten it wrong.